### PR TITLE
Release 35.0.0

### PR DESCRIPTION
**Pull request by hmalik88**

### CODE REVIEW

**Review by legobeat — APPROVED** [review]
lgtm

File: packages/base-controller/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+### Added
+- `update` function updated to expose patches, added `applyPatches` function to BaseControllerV2 ([#980](https://github.com/MetaMask/controllers/pull/980))
```
Comment (mcmire) [inline packages/base-controller/CHANGELOG.md:11]: Changelog entries are allowed to be split up -- in other words, there doesn't have to be a 1:1 correlation between entries and PR. One of these is a change, but the other is an addition. What do you think about splitting this entry up into both the Changed and Added sections?

Also (this is a nit) it's a convention to use the imperative for these entries (so, "add" vs. "added") as well as the active tense ("update `update` to..." vs. "`update` updated to...").

**Review by mcmire — COMMENTED** [review]
It seems like we could tweak the changelog entry a bit, but other than that this looks good to me.

File: packages/base-controller/CHANGELOG.md (hunk 1 above)
Comment (hmalik88) [reply packages/base-controller/CHANGELOG.md]: Sounds good to me, I just split into Changed and Added, also updated the language.

**Comment by Gudahtt** [comment]
>Make sure when you squash and merge that the name of the commit is "Release 35.0.0" and not "Release/35.0.0". The commit message needs to match a specific format in order for the release to be published.

I have just updated the title to be "Release 35.0.0" to make this simpler (it should now default to the correct subject line).

It seems that if you create the PR with just a single commit, the default is correct. But if you update the changelog before creating the PR, it uses the branch name rather than the first commit message as the default PR title.

We can introduce a lint to catch this in the future.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by mcmire — APPROVED** [review]
LGTM as well.

**Pull request merged**